.travis-ci/deploy.ps1

```diff
@@ -0,0 +1,47 @@
+#!/bin/pwsh
+$ErrorActionPreference = 'Stop'
+
+$TimeStampString = (Get-Date).ToString('ddd MMM d HH:mm:ss K')
+$VscodePath = Join-Path 'public' 'vscode'
+$LinguaPath = Join-Path 'public' 'lingua-pupuli'
+$GithubToken = $ENV:GITHUB_TOKEN
+
+if ($null -eq $GithubToken) {
```

Comment:
SHould this be `$env:GithubToken`?

Reply:
Nope. Default on non-Windows platforms is uppercase env vars.

Reply:
No, I was being dumb. I missed line 7 and thought you forgot `$env:`

Reply:
No I was being dumb. I missed line 7